test/simulation/tests-security.go

```diff
@@ -268,37 +271,39 @@ func testMakerShutdownAfter1stHTLC(net *xudtest.NetworkHarness, ht *harnessTest)
 	ht.assert.Equal(len(bobChan.PendingHtlcs), 1)
 	expirationHeight := bobChan.PendingHtlcs[0].ExpirationHeight
 
-	// Try to expire the HTLC.
+	// Expire the HTLC.
 
 	ltcInfo, err := net.LndLtcNetwork.LtcMiner.Node.GetInfo()
 	ht.assert.NoError(err)
-	_, err = net.LndLtcNetwork.LtcMiner.Node.Generate(expirationHeight - uint32(ltcInfo.Blocks) + 100)
+	_, err = net.LndLtcNetwork.LtcMiner.Node.Generate(expirationHeight - uint32(ltcInfo.Blocks))
 	ht.assert.NoError(err)
 	ltcInfo, err = net.LndLtcNetwork.LtcMiner.Node.GetInfo()
-	ht.assert.True(ltcInfo.Blocks > int32(expirationHeight))
+	ht.assert.True(ltcInfo.Blocks >= int32(expirationHeight))
 
-	err = net.Bob.LndLtcNode.WaitForBlockchainSync(ht.ctx)
-	ht.assert.NoError(err)
-	err = net.Alice.LndLtcNode.WaitForBlockchainSync(ht.ctx)
-	ht.assert.NoError(err)
+	time.Sleep(25 * time.Second)
 
-	// Attempt to close the channel cooperatively.
+	// Verify channel was closed (by force-close, as the outgoing HTLC got expired).
 
-	err = closeLtcChannel(ht.ctx, net.LndLtcNetwork, net.Bob.LndLtcNode, bobLtcChanPoint, false)
-	ht.assert.EqualError(err, "rpc error: code = Unknown desc = cannot co-op close channel with active htlcs")
+	bobLtcChanList, err = net.Bob.LndLtcNode.ListChannels(ht.ctx, &lnrpc.ListChannelsRequest{})
+	ht.assert.Equal(len(bobLtcChanList.Channels), 0)
 
-	// Close the channel uncooperatively.
+	// Wait for publishing CLTV-delayed HTLC output using timeout tx.
 
-	err = closeLtcChannel(ht.ctx, net.LndLtcNetwork, net.Bob.LndLtcNode, bobLtcChanPoint, true)
+	_, err = net.LndLtcNetwork.LtcMiner.Node.Generate(1)
 	ht.assert.NoError(err)
 
-	//_, err = net.LndLtcNetwork.LtcMiner.Node.Generate(1100)
-	//ht.assert.NoError(err)
-	//err = net.Bob.LndLtcNode.WaitForBlockchainSync(ht.ctx)
-	//ht.assert.NoError(err)
-	//
-	//time.Sleep(5 * time.Second)
+	time.Sleep(5 * time.Second)
+
+	// Wait for HTLC output to be spentfully confirmed.
```

Comment:
"Spentfully" is a new word for me :stuck_out_tongue:. I'm guessing this means to just say "to be fully confirmed"?

Reply:
I'm not familiar with it as well ;)
Will fix.